pkg/cmd/attestation/api/client.go

```diff
@@ -130,6 +167,70 @@ func (c *LiveClient) getAttestations(url, name, digest string, limit int) ([]*At
 	return attestations, nil
 }
 
+func (c *LiveClient) fetchBundlesByURL(attestations []*Attestation) ([]*Attestation, error) {
+	fetched := make([]*Attestation, len(attestations))
+	g := errgroup.Group{}
+	for i, a := range attestations {
+		g.Go(func() error {
+			b, err := c.fetchBundleByURL(a)
+			if err != nil {
+				return fmt.Errorf("failed to fetch bundle with URL: %w", err)
+			}
+			fetched[i] = &Attestation{
+				Bundle: b,
+			}
+			return nil
+		})
+	}
+
+	if err := g.Wait(); err != nil {
+		return nil, err
+	}
+
+	return fetched, nil
+}
+
+func (c *LiveClient) fetchBundleByURL(a *Attestation) (*bundle.Bundle, error) {
+	// for now, we fallback to the bundle field if the bundle URL is empty
+	if a.BundleURL == "" {
+		c.logger.VerbosePrintf("Bundle URL is empty. Falling back to bundle field\n\n")
+		return a.Bundle, nil
+	}
+
+	c.logger.VerbosePrintf("Fetching attestation bundle with bundle URL\n\n")
+
+	resp, err := c.httpClient.Get(a.BundleURL)
+	if err != nil {
+		return nil, err
+	}
+
+	defer resp.Body.Close()
+	if resp.StatusCode != http.StatusOK {
+		return nil, fmt.Errorf("unexpected status code: %d", resp.StatusCode)
+	}
+
+	// parse the URL to get the host and path
```

Comment:
What is this comment referencing?

Reply:
Just some leftover comment that should be deleted.

Reply:
This was just an accidental copy paste.